lib/iris/coord_systems.py

```diff
@@ -840,17 +841,22 @@ def __init__(self, longitude_of_projection_origin=0, ellipsoid=None):
                     True longitude of planar origin in degrees.
             * ellipsoid
                     :class:`GeogCS` defining the ellipsoid.
+            * scale_factor_at_projection_origin
+                    the latitude where the scale is 1. Defaults to 0 degrees.
 
         """
-
-        #: True longitude of planar origin in degrees.
```

Comment:
I agree that these aren't particularly well written, but I don't want to *lose* the docstrings altogether. 
e.g. https://scitools.org.uk/iris/docs/latest/iris/iris/coord_systems.html#iris.coord_systems.Mercator.ellipsoid

Reply:
Sorry! I _always_ forget about these special comments (perhaps because it's so unclear that they do even have special behaviour). In fact, I thought they were just unnecessary duplications of the docstring above...

I'll reinstate them and add another for the new kwarg I've introduced.